### DISCUSSION TITLE
How to determine the quality of the result of each epoch

### DISCUSSION THREAD

**Discussion by WillIsback** [JarodMica/ai-voice-cloning#47] [Q&A] [answered]
Dear JarodMica,

First I want to thank you for all the good stuff you produced it's such a gold mine. I originaly plan to train my own japanese model with moe-speech dataset but before the big jump I made some test on little scale. Training a 10mn of audio file with 500 epoch.

My questions is how to determine the quality result of each epochs ? I saw on internet that you can somehow determine it by comparing your training score and your validation score. But I have not check validation and even if I have on such a small dataset it is not relevent.

Please enlight me on how did you determine your best model after training.

Best regards, wish you the best for the futur !

**Comment by JarodMica** [comment, marked as answer]
Epoch loss values I have found tends to mean very little for tortoise, however, it should be noted, that if the text or mel lines go to 0 loss on the graph, most likely the model will sound pretty terrible.

The best way to determine a good model after training is through listening to it.  I haven't found any positive correlations between values and outputted models unfortunately.
